README.md

```diff
@@ -3,6 +3,7 @@ Boost::ASIO low-level redis client (connector)
 
 [![Travis](https://img.shields.io/travis/basiliscos/cpp-bredis.svg)]()
 [![license](https://img.shields.io/github/license/basiliscos/cpp-bredis.svg)]()
+[![Coveralls](https://img.shields.io/coveralls/basiliscos/cpp-bredis.svg)]()
```

Comment:
Seems it takes tests in `t` directory too, I'll fix that later. 

The most of code has been refactored to handle new interfaces. Now it works even faster then before, because it just marks up input stream, which byitself stays untouched!

Right now I'm looking at coroutines/futures support.